evap/staff/views.py

```diff
@@ -1386,6 +1386,36 @@ def evaluation_email(request, evaluation_id):
     )
 
 
+def import_or_copy_participants(request, operation, import_type, evaluation, copy_form) -> bool:
+    if "test" in operation:
+        return False
```

Comment:
This should not be possible given the calling code

Reply:
yes. Should this become an assertion instead?

Reply:
(resolved with other points)